docs/guides/dynamo_run.md

```diff
@@ -111,6 +112,43 @@ The `llama3B_pool` name is purely symbolic, pick anything as long as it matches
 
 Run `dynamo-run --help` for more options.
 
+### KV-aware routing
+
+**Setup**
+
+Only patched vllm currently supports KV-aware routing. Key setup steps:
+
+1. `etcd` and `nats` (see earlier) must be running and accessible from all nodes.
+1. Create a virtualenv: `uv venv kvtest`, source it's `activate`.
```

Comment:
Nit: 1,2,3,4,5,6 over 1,1,1... Even though it renders fine

Reply:
The repeated `1.` is a markdown numbered list. It allows you to move things around in the list without updating all the numbers.

I'm OK changing it if you prefer though.

Reply:
Not strongly enough to matter, I just think it trips people up when they're following steps from `cat README.md` in a shell or something similar, but no big deal, resolving